packages/accordions/src/styled/StyledLabel.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.step_label';
+
+interface IStyledLabelProps {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledLabel = styled.div.attrs<IStyledLabelProps>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledLabelProps>`
+  display: ${props => !props.isHorizontal && 'flex'};
+  align-items: ${props => !props.isHorizontal && 'center'};
+  text-align: ${props => props.isHorizontal && 'center'};
+  line-height: ${props => getLineHeight(props.theme.lineHeights.lg, props.theme.fontSizes.md)};
```

Comment:
Do not use `theme.lineHeights`; use `space.base` instead. Also, designs call for the label to be the standard line-height of 20px, not 24px. This won't display properly when the line wraps. You'll probably need to apply padding to make up for the difference between the 20px line-height and the 24px icon to which this is intended to be centered.